### PR TITLE
Fix generation of float numbers

### LINKED ISSUE #401: random.number precision option needs improvement
using this code:

`var faker = require("Faker")
var options = {min: 0, max: 10, precision:1}
options.precision
faker.random.number(options)`

i would expect a number between 0 and 10 to be returned, but i just get NaN. looking at the code this is what i would expect to happen.
I am simply trying to understand the methods in order to generate a float, there doesnt appear to be much documentation around these methods.

Does anyone have any idea what i am doing wrong?

### PR DESCRIPTION
**Pull request by IvanGoncharov**
Fixes #401

### CODE REVIEW

File: lib/random.js [hunk 1]
```diff
@@ -47,8 +47,10 @@ function Random (faker, seed) {
       max += options.precision;
     }
 
-    var randomNumber = options.precision * Math.floor(
+    var randomNumber = Math.floor(
       mersenne.rand(max / options.precision, options.min / options.precision));
+    // Workaround problem in Float point arithmetics for e.g. 6681493 / 0.01
+    randomNumber /= 1 / options.precision;
```
Comment (Marak) [inline lib/random.js]: Could you possibly expand this out to make it easier to read?

I'm not actually certain what this is doing.

Thanks!

**Comment by Marak** [comment]
@IvanGoncharov -

Looks good. Could you adjust the code to be a bit more easier to understand? I made a comment.

Thank you for your contribution.

**Comment by IvanGoncharov** [comment]
@Marak Done

**Comment by Marak** [comment]
Thanks @IvanGoncharov, very nice!

**Comment by IvanGoncharov** [comment]
@Marak My pleasure. Can you please publish npm package with this fix?

**Comment by Marak** [comment]
I'll do a patch version in an hour or two.

**Comment by IvanGoncharov** [comment]
@Marak Thank you.

**Pull request merged**